.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish a new release
+
+on:
+  push:
+    branches:
+    - feat/JsonAdapter
```

Comment:
Is this really useful to release only on one specific branch ?

Reply:
Mb, it was supposed to be 'main' here